src/delay_handler.cc

```diff
@@ -7,6 +7,7 @@ using std::string;
 
 RequestHandler::Status DelayHandler::Init(const std::string& uri_prefix, const NginxConfig& config) {
   delay_ = DEFAULT_DELAY;
+  (void) uri_prefix; // suppress unused parameter warning
```

Comment:
I don't remember, can you suppress this by removing the name from the parameter?
 i.e. `Delayhandler::Init(const std::string&, const NginxConfig& config)`